src/Orchard.Web/Modules/Orchard.Taxonomies/Services/TaxonomyService.cs

```diff
@@ -30,7 +30,7 @@ public class TaxonomyService : ITaxonomyService {
         private readonly ShellSettings _shellSettings;
         private readonly IShellDescriptorManager _shellDescriptorManager;
 
-        private readonly HashSet<int> _processedTermParts = new HashSet<int>(); 
+        private readonly HashSet<int> _processedTermParts = new HashSet<int>();
```

Comment:
"_processedTermPartIds" shows better what it's used for.

Reply:
I think I only changed the new line here, by reformatting the file, because I don't remember using this. I agree with you though